model/src/main/kotlin/LicenseFinding.kt

```diff
@@ -86,18 +86,28 @@ class LicenseFindingDeserializer : StdDeserializer<LicenseFinding>(LicenseFindin
         return when {
             node.isTextual -> LicenseFinding(node.textValueOrEmpty(), sortedSetOf(), sortedSetOf())
             else -> {
+                val copyrightsType = jsonMapper.typeFactory.constructCollectionType(
+                        TreeSet::class.java,
```

Comment:
This code is redundant. How about extracting it to constructTreeSetType(clazz: java.lang.class).
You could get rid of the copyrightsType variable and safe 3 x 4 lines

Reply:
Done.